lib/jelix/plugins/formwidget/time_html/time_html.formwidget.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * @see        http://www.jelix.org
+ * @licence     http://www.gnu.org/licenses/lgpl.html GNU Lesser General Public Licence, see LICENCE file
+ */
+
+/**
+ * HTML form builder.
+ *
+ * @see http://developer.jelix.org/wiki/rfc/jforms-controls-plugins
+ */
+class time_htmlFormWidget extends \jelix\forms\HtmlWidget\WidgetBase
+{
+    protected function outputJs()
+    {
+        $ctrl = $this->ctrl;
+        $jFormsJsVarName = $this->builder->getjFormsJsVarName();
+        $js = 'c = new '.$jFormsJsVarName."ControlTime('".$ctrl->ref."', ".$this->escJsStr($ctrl->label).");\n";
+        $js .= "c.multiFields = true;\n";
+        $minTime = $ctrl->datatype->getFacet('minValue');
+        $maxTime = $ctrl->datatype->getFacet('maxValue');
+        if ($minTime) {
+            $js .= "c.min${minTime} = '".$minTime->toString(jDateTime::DB_TFORMAT)."';\n";
+        }
+        if ($maxTime) {
+            $js .= "c.max${maxTime} = '".$maxTime->toString(jDateTime::DB_TFORMAT)."';\n";
+        }
+        $this->parentWidget->addJs($js);
+        $this->commonJs();
+
+        if ($ctrl instanceof jFormsControlTime || get_class($ctrl->datatype) == 'jDatatypeTime' || get_class($ctrl->datatype) == 'jDatatypeLocaleTime') {
+            $config = isset($ctrl->timepickerConfig) ? $ctrl->timepickerConfig : jApp::config()->forms['timepicker'];
+            $this->parentWidget->addJs('jelix_timepicker_'.$config."(c, jFormsJQ.config);\n");
```

Comment:
we don't have a default timepicker provided by a lib like jQueryUI, so we should do this only if `$config` is not empty